vaadin-app-layout-flow-parent/vaadin-app-layout-flow/src/main/java/com/vaadin/flow/component/applayout/AppLayout.java

```diff
@@ -284,4 +363,36 @@ public static Section fromWebcomponentValue(String webcomponentValue) {
                     : null;
         }
     }
+
+    /**
+     * The internationalization properties for {@link AppLayout}
+     */
+    public static class AppLayoutI18n implements Serializable {
+        private String drawer;
+
+        /**
+         * Gets the text for the `aria-label` attribute on the drawer. The
+         * attribute is set when the drawer is in the overlay mode and announced
+         * once the drawer is opened.
+         *
+         * @return the drawer aria-label
+         */
+        public String getDrawer() {
+            return drawer;
+        }
+
+        /**
+         * Sets the text for the `aria-label` attribute on the drawer. The
+         * attribute is set when the drawer is in the overlay mode and announced
+         * once the drawer is opened.
```

Comment:
I think it's enough to have the explanation on either the getter or the setter.

Reply:
Left for the setter, removed for the getter.